src/framework/components/anim/system.js

```diff
@@ -0,0 +1,76 @@
+Object.assign(pc, function () {
+    var _schema = [
+        'enabled',
+        'speed',
+        'activate'
+    ];
+
+    /**
+     * @class
+     * @name pc.AnimComponentSystem
+     * @augments pc.ComponentSystem
+     * @classdesc The AnimComponentSystem manages creating and deleting AnimComponents.
+     * @description Create an AnimComponentSystem.
+     * @param {pc.Application} app - The application managing this system.
+     */
+    var AnimComponentSystem = function AnimComponentSystem(app) {
+        pc.ComponentSystem.call(this, app);
+
+        this.id = 'anim';
+        this.description = "State based animation system that can animate the models and component properties of this entity and its children";
+
+        this.ComponentType = pc.AnimComponent;
+        this.DataType = pc.AnimComponentData;
+
+        this.schema = _schema;
+
+        this.on('beforeremove', this.onBeforeRemove, this);
+        this.on('update', this.onUpdate, this);
+
+        pc.ComponentSystem.bind('update', this.onUpdate, this);
+    };
+    AnimComponentSystem.prototype = Object.create(pc.ComponentSystem.prototype);
+    AnimComponentSystem.prototype.constructor = AnimComponentSystem;
+
+    pc.Component._buildAccessors(pc.AnimComponent.prototype, _schema);
+
+    Object.assign(AnimComponentSystem.prototype, {
+        initializeComponentData: function (component, data, properties) {
+            properties = ['activate', 'enabled', 'speed'];
+            pc.ComponentSystem.prototype.initializeComponentData.call(this, component, data, properties);
+        },
+
+        cloneComponent: function (entity, clone) {
```

Comment:
You don't need to override cloneComponent if you're just copying over properties I think. I believe you only need to override it if you have any Object-type properties like arrays or objects that need to be deep copied.

Reply:
Removed this.